packages/uikit/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "./src",
```

Comment:
Hmm. We are going to use the same tsconfig in `packages` folder when we are moving to `monorepo`. Not sure if this condig is still compatible with it

Reply:
Ok, I think I can use relative import for now